catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswTransformProvider.java

```diff
@@ -84,12 +85,12 @@ public void marshal(Object o, HierarchicalStreamWriter writer, MarshallingContex
             return;
         }
         Metacard metacard = (Metacard) o;
-        Object arg = context.get(CswConstants.OUTPUT_SCHEMA_PARAMETER);
+        String keyArg = (String)context.get(CswConstants.TRANSFORMER_LOOKUP_KEY);
```

Comment:
The marshal & unmarshal should be consistent. If you are going to change marshal to use these keys then you need to update unmarshal to behave the same.